generators/app/files/CONTRIBUTING.md

```diff
@@ -2,9 +2,123 @@
 
 Please follow these guidelines for contributing to this repository.
 
+## Branches
+
+**Never commit directly to `master`.** Create small branches for *each* topic that you work on
+(individual features, bug fixes, and so on). When the branch is complete,
+[create a pull request](https://help.github.com/articles/using-pull-requests/) against `master`.
```

Comment:
Never commit directly to `master` _or other long-running branches_. (e.g. a production branch).